src/cargo/util/shell_escape.rs

```diff
@@ -0,0 +1,115 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#[cfg(target_os = "windows")]
+pub use self::windows::shell_escape;
+
+
+#[cfg(any(test, target_os = "windows"))]
```

Comment:
er actually I was thinking that this would remove the `#[cfg]` attributes in this module entirely, and instead have a top-level function that used `if cfg!(...)`